### ISSUE TITLE
voilaexporter has no attribute contents_manager for default template

### ISSUE THREAD

**Issue by richtong** [voila-dashboards/voila#689]
We are tearing our hair out here. We have a working configuration of Voila running Vuetify. The underlying Dashboard displays fine with Voila. The very strange thing is that we have all dependencies pinned so it seems unlikely to be a version problem, but then if you sometimes you get this error and sometimes you don't.

We think there must be something changing templates outside of our github repo. You seem to copy things into ~/share in case our pipenv, so maybe these are somehow changing? IN any case this does not work

`voila file.ipynb` using default but `voila --template=vuetify_default file.ipynb` does work. It seems intermittent. In that even a clean install will sometimes show the problem and sometimes not.

```
  File "/Users/rich/.local/share/virtualenvs/nb-FpFnjW0W/lib/python3.8/site-packages/nbconvert/exporters/../templates/skeleton/null.tpl", line 81, in block "any_cell"
    {%- block markdowncell scoped-%} {%- endblock markdowncell -%}
  File "/Users/rich/.local/share/virtualenvs/nb-FpFnjW0W/share/jupyter/voila/templates/default/nbconvert_templates/lab.tpl", line 92, in block "markdowncell"
    {{ cell.source  | markdown2html | strip_files_prefix }}
  File "/Users/rich/.local/share/virtualenvs/nb-FpFnjW0W/lib/python3.8/site-packages/voila/exporter.py", line 55, in markdown2html
    contents_manager=self.contents_manager,
'VoilaExporter' object has no attribute 'contents_manager'
```

We have seen this error before and pinned jupyter-server to 0.1.1 and this seems to help  but that is already set

We are running bqplot 0.12.16, voila-vuetify==0.4.01 and voila 0.1.22

**Comment by gbrault** [comment]
I have a similar issue, just loading ipywidgets in voila:

File "/home/gilbert/venv/lib/python3.8/site-packages/voila/exporter.py", line 55, in markdown2html

**context**

[piplist.txt](https://github.com/voila-dashboards/voila/files/5160763/piplist.txt)

    contents_manager=self.contents_manager,
AttributeError: 'VoilaExporter' object has no attribute 'contents_manager'

**Comment by SylvainCorlay** [comment]
cc @maartenbreddels

**Comment by maartenbreddels** [comment]
looking into this.
@richtong which version of nbconvert are you using?

**Comment by SylvainCorlay** [comment]
This appears to be due to the recent release of traitlets 5.0.0, which is backward incompatible.

**Comment by SylvainCorlay** [comment]
@richtong @gbrault the solution for now is to pin traitlets to be `<5`.

**Comment by gbrault** [comment]
Bravo @SylvainCorlay `pip install traitlets==4.3.3` works just fine!

**Comment by SylvainCorlay** [comment]
I am pushing for a new patch release with that fix!

**Comment by maartenbreddels** [comment]
Really good catch @SylvainCorlay!

**Comment by maartenbreddels** [comment]
This is fixed and released for v0.1.23 and in also fixed in master.

**Comment by richtong** [comment]
Thanks much!

Sent via Superhuman iOS ( https://sprh.mn/?vip=rich@tongfamily.com )
